pyproject.toml

```diff
@@ -147,7 +147,7 @@ select = [
 ]
 
 [tool.pytest.ini_options]
-addopts = "-m 'not integration and not performance and not e2e and not spcs and not loaded_modules and not integration_experimental'"
+addopts = "-vv --maxfail=10 -m 'not integration and not performance and not e2e and not spcs and not loaded_modules and not integration_experimental'"
```

Comment:
`--maxfail=10` - do we want to keep it? Do I correctly understand that in case of less failures, the final result will still a general failure?

Reply:
Yes. When we discussed this few days ago we agreed that if too many test are failing it means a wide spread problem. Using random order makes this indeterministic but still there's a small optimization for changes that are not well tested.